src/components/Header/Header.js

```diff
@@ -1,23 +1,35 @@
 import React from 'react'
 // import ash from '/Images/ash1.png
 import './Header.css'
+import { Route, Link } from 'react-router-dom';
 //Assets/Images/ash1.png
+
+
 const Header = () => {
   return (
     <header className='header'>
 
       <nav className='nav'>
         <ul className='nav-links'>
-          <li className='how-to'>How To</li>
+          <Link
+            to={`/help`}
+            key={`help`}  
+          >
+            <li className='how-to'>How To</li>
+          </Link>
+
```

Comment:
Router! Not that it's needed, but this could be a good use case for NavLink. Just a suggestion though, Link is great too :)

Reply:
I really like the Link here!